tests/test_reader.py

```diff
@@ -0,0 +1,54 @@
+import csv
+
+import pytest
+from mock import Mock
+
+from datarobot_batch_scoring.reader import FastReader, SlowReader
+
+
+class TestCSVReaderWithWideData(object):
```

Comment:
Let's keep our test structure somewhat consistent and don't use unittest-like classes and put fixtures in conftest and use plain `test_` in test files.